README.md

```diff
@@ -1,105 +1,80 @@
+![code_flow](./figs/bmi_pet.png)
+
+# Notes from author of the PET functions
+evapotranspiration (ET) module,  
+Version 1.0 by Fred L. Ogden, NOAA-NWS-OWP, May, 2020.  
+includes five different methods to clculate ET, from Chow, Maidment & Mays Textbook, and UNFAO Penman-Monteith:  
+1. energy balance method
+2. aerodynamic method
+3. combination method, which combines 1 & 2.
+4. Priestley-Taylor method, which assumes the ratio between 1 & 2, and only calculates 1.
+5. Penman-Monteith method, which requires a value of canopy resistance term, and does not rely on 1 or 2.
+This subroutine requires a considerable amount of meteorological data as input.
+   * a) temperature and (relative-humidity or specific humidity) and the heights at which they are measured.
+   * b) near surface wind speed measurement and the height at which it was measured.
+   * c) the ambient atmospheric temperature lapse rate
+   * d) the fraction of the sky covered by clouds
+   * e) (optional) the height above ground to the cloud base. If not provided, then assumed.
+   * f) the day of the year (1-366) and time of day (UTC only!)
+   * g) the skin temperature of the earth's surface, TODO: should come from another module to calc. soil or veg. temp.
+   * h) the zero-plane roughness height of the atmospheric boundary layer assuming log-law behavior (from land cover)
+   * i) the average root zone soil temperature, or near-surface water temperature in the case of lake evaporation.
+   * j) the incoming solar (shortwave) radiation.  If not provided it is computed from d,e,f, using an updated method similar to the one presented in Bras, R.L. Hydrology.  Requires value of the Linke atmospheric turbidity factor, which varies from 2 for clear mountain air to 5 for smoggy air.  According to Hove & Manyumbu 2012, who calculated values over Zimbabwe that varied from 2.14 to 3.71.  Other values exist in the literature. TODO: This turbidity factor could be calculated from satellite obs. or maybe NOAA already does this? All radiation calculations needed for 1, 3, 4, and 5 require net radiation calculations at the land surface. The net radiation is calculated using a, c, d, e, f, g, j, plus the Linke turbidity factor, which can be estimate from satellite observations.  
```

Comment:
typo: estimated

Reply:
Typo removed since moving the TODO to an issue. Thanks.